buildtools/build_wxwidgets.py

```diff
@@ -496,6 +496,9 @@ def main(wxDir, args):
                         args[i] = '-D' + arg
                     buildDir = wxRootDir
                     nmakeCommand = 'cmake.exe'
+                    os.environ['INCLUDE'] += (
+                        ';' + os.path.join(wxRootDir, 'include', 'msvc')
```

Comment:
you say ";" is `os.pathsep` ?

Reply:
yes it is and I should have used os.pathsep. didn't think about it tho